WalletWasabi/Extensions/VerboseTransactionInfo.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+
+namespace NBitcoin.RPC
```

Comment:
Don't put these model classes to the `Extensions` folder. Place them to `BitcoinCore` or the `Blockchain` folder coupled together into a new folder. Also make sure namespaces are correct.

Reply:
ah good point ok let me do that next